src/System.Private.Xml/src/System.Private.Xml.builds

```diff
@@ -3,6 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Private.Xml.csproj" />
+    <Project Include="System.Private.Xml.csproj">
+      <TargetGroup>netcore50</TargetGroup>
+    </Project>
+    <Project Include="System.Private.Xml.csproj">
+      <TargetGroup>netcore50aot</TargetGroup>
+    </Project>
```

Comment:
In the src System.Private.Xml.csproj, we have this line:
`<DefineConstants Condition="'$(TargetGroup)' == 'netcore50aot'">$(DefineConstants);NET_NATIVE</DefineConstants>`
which requires it to build a slightly different code for netcore50aot in some cases.